admin-purchase-options-action/package.json.liquid

```diff
@@ -0,0 +1,27 @@
+{%- if flavor contains "react" -%}
+{
+  "name": "{{ handle }}",
+  "private": true,
+  "version": "1.0.0",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "react": "^18.0.0",
+    "@shopify/ui-extensions": "unstable",
+    "@shopify/ui-extensions-react": "unstable"
```

Comment:
These should be pointing to the latest stable calver (10-24) not to unstable

Reply:
Thanks~ Will change.